Multi-engine renaming (#471)
OpenMDAO/Aviary: aviary/docs/misc_resources/glossary.md

flight envelope
    The limits of speed, altitude, and load factor that an aircraft can safely operate within. Understanding the flight envelope is critical for aircraft design and operation.

homogeneous engines

>     Engines that are assumed to have the same performance characteristics.

Reviewer: ```suggestion
    An individual engine or set of engines that all have identical performance characteristics.
```

    E.g. you might have four engines on an aircraft, but they are all assumed to have the same thrust, fuel consumption, etc.

>     Each of these engines would use the same `EngineModel` object in Aviary.

Reviewer: ```suggestion
    All engines in a homogenous set use the same `EngineModel` object in Aviary.
```

    See also; heterogeneous engines.

heterogeneous engines

>     Engines that are assumed to have different performance characteristics.

Reviewer: ```suggestion
    Multiple sets of homogenous engines, with each set having unique performance characteristics.
```

>     This is useful if you are modeling an aircraft with engines of different sizes or types.

Reviewer: ```suggestion
    This is useful if you are modeling an aircraft with engines of different sizes or types.
    Each unique engine would use the same `EngineModel` object in Aviary.
```
